test/decl/class/override.swift

```diff
@@ -160,6 +160,26 @@ class H : G {
   override init(b: Double) {} // expected-error{{initializer does not override a designated initializer from its superclass}} {{none}}
 }
 
+class J {
+  func f1(_: Int, int1: Int) { } // expected-note{{potential overridden instance method 'f1(_:int1:)' here}} {{26-26=, int1: Int}}
```

Comment:
Question... the fix-it shouldn't be on the override function declaration? I mean what we want is to add parameters into the `override func f1(_: Int) { }` child method declaration in order to match with the parent `func f1(_: Int, int1: Int) { }` or am I understanding it wrong?

Reply:
I think add parameters here don't seem right ...

Reply:
Yeah, the code in the is exactly what we expected, note that the fix-it is on the `override` method which seems to be the correct. So in this case for example inserting this `, int1: Int` here on the parent definition wouldn't make sense because what we want with this fix-it is to make the override method to match the parent declaration, so the correct would probably be inserting on line 176 where the fix-it would make `override func f1(_: Int) { }` become `override func f1(_: Int, int1: Int) { }` which will fix the method in order to match the parent definition. If I understood correctly that is the intended here.

Reply:
I get it, and fix it with reference to [test/Constraints/diag_missing_arg.swift](https://github.com/apple/swift/blob/9af806e8fd93df3499b1811deae7729176879cb0/test/Constraints/diag_missing_arg.swift)